sensors/aidl/Sensor.cpp

```diff
@@ -360,6 +462,29 @@ std::vector<Event> OnChangeSensor::readEvents() {
     return outputEvents;
 }
 
+void Sensor::setAdditionalInfoFrames() {
+    // https://developer.android.com/reference/android/hardware/SensorAdditionalInfo#TYPE_SENSOR_PLACEMENT
+
```

Comment:
We use the same code base for CIV as well, and additional info may not be needed for CIV.
Can we add this code in condition based on TARGET_PRODUCT?

Reply:
yes this can be done to support target specific features.